editor/utils/url.js

```diff
@@ -23,7 +23,8 @@ export function getPostEditUrl( postId ) {
  * @return {string} WPAdmin URL.
  */
 export function getWPAdminURL( page, query ) {
-	return addQueryArgs( page, query );
+	const url = ( window._wpAdminURL ) ? window._wpAdminURL + page : page;
```

Comment:
I think we should avoid introducing new globals, instead, I'd personally prefer something like we're doing in #7018 :

A config function `setWPAdminUrl()` and call this function using `wp_add_inline_script` when registering the `wp-utils` script on the server.